docs/man/docker-run.1.md

```diff
@@ -528,9 +528,9 @@ colon:
 
     # docker run -v /var/db:/data1 -i -t fedora bash
 
-When using SELinux, be aware that the host has no knowledge of container SELinux
+When using SELinux, be aware that the container has no knowledge of host SELinux
```

Comment:
You should state the problem immediately after the example -- the example context is clear given the position of the sentence and the references.  To bits of information are important the fact that the **host** is runing SELinux and the container cannot write. If you state these two ideas at the first and last bit of the sentence you leverage the two "power" ends of a sentence the beginning and the end.

You should define SELInux as it is the first occurence on this page. 

In your work around, you don't need the "at the time of this" clause.  That fact is assumed by the reader.  It isn't clear when or where for running the `chcon` command. It is an SELinux command so --- run it on the host and state this clearly.  Does the user need to set the context before mounting --- not clear but I assumed that the answer was yes.  
-----------

If your host is using SELinux (Security-Enhanced Linux),  you can mount the `/var/db` directory but the container cannot write to `/data1`.  Instead, attempts to write to `/data1` result in a  `Permission denied` message. This is because the container has no knowledge of the host's SELinux policies. 

To work around this, change the security context on the `/var/db` directory. This command applies an appropriate SELINUX policy to the directory:
 
 # chcon -Rt svirt_sandbox_file_t /var/db

Then,  rerun the container; the `/data1` is now writable.